scripts/run_backend_tests.py

```diff
@@ -227,10 +233,14 @@ def _get_test_target_classes(path):
 def main(args=None):
     """Run the tests."""
     parsed_args = _PARSER.parse_args(args=args)
+    # These environmental variables are required to allow Google Cloud Tasks to
+    # operate in a local development environment without connecting to the
+    # internet. These environment variables allow Cloud APIs to be instantiated.
+    os.environ['CLOUDSDK_CORE_PROJECT'] = 'oppia-dev'
```

Comment:
Can you put this in scripts/common.py perhaps? Don't repeat the same stuff everywhere.

Ditto for anything you repeat in multiple scripts -- if you have it repeated all over the place, it's quite hard to maintain.